### PR TITLE
Make install.sh and uninstall.sh idempotent

### LINKED ISSUE #12: Install.sh is not idempotent
**Description:**
Currently, install.sh starts the program without checking if the program has already been started.

**Steps to reproduce:**
1) Run install.sh
2) Run install.sh (second time)

**Results:**
The indicator is displayed twice

**Expected result:**
The indicator is only started the first time.

### PR DESCRIPTION
**Pull request by yorickvanzweeden**
Fixes #12 

What I did:
- Make `install.sh` idempotent. This makes script idempotent, and thus more easily automate-able using software like Chef, Puppet, or Ansible
-  Encapsulate echo contents with quotes. This improves code readability in editors like nano
- Install.sh only installs dependencies when they are not present
- Uninstall.sh offers to remove dependencies

### CODE REVIEW

**Comment by yorickvanzweeden** [comment]
@ilcardella Could you have a look at my PR? Seems fair that I get reviewed too ;)

File: install.sh [hunk 1]
```diff
@@ -1,59 +1,74 @@
...
+
+# Install indicator
 install_indicator
 
 # Starting script
-echo Starting indicator
-nohup $(command -v python) /opt/ubuntu-nordvpn-indicator/nordvpn_indicator.py >/dev/null 2>&1 &
+if pgrep -f "nordvpn_indicator" > /dev/null
+then
+    echo "Indicator already running"
```
Comment (ilcardella) [inline install.sh]: I would kill the process and then still start the new version that has just been installed. What do you think?

File: uninstall.sh [hunk 2]
```diff
@@ -1,16 +1,33 @@
 #!/bin/bash
 
-echo Removing Ubuntu NordVPN Indicator
+echo "Removing Ubuntu NordVPN Indicator"
 rm ~/.config/autostart/ubuntu-nordvpn-indicator.desktop
 sudo rm -rf /opt/ubuntu-nordvpn-indicator
 
+read -p "Do you want to uninstall AppIndicator? " -n 1 -r
```
Comment (ilcardella) [inline uninstall.sh]: What about adding `[Y/n]` at the end of the message to make it clear what is the expected input?

Comment (ilcardella) [reply uninstall.sh]: Same for the messages below

**Review by ilcardella — CHANGES_REQUESTED** [review]
Last comment. I would remove all the `sudo` keyword before each command. The caller would invoke the script as root in the best way he/she think.

We could even check at the very beginning of both scripts if the current user is root, and if not, exit with an error code and a message about requiring root permission

**Comment by yorickvanzweeden** [comment]
@ilcardella I wouldn't call sudo on entire scripts, as it could mess with permissions. For example: if you do a `cp`, it would run as root, and the resulting file would have root permissions. The resulting file can't be moved by normal users anymore.

We do require sudo for apt commands

File: install.sh (hunk 1 above)
Comment (yorickvanzweeden) [reply install.sh]: Yes, seems good. See newest commit

File: uninstall.sh (hunk 2 above)
Comment (yorickvanzweeden) [reply uninstall.sh]: Yes, seems good. See newest commit

**Review by ilcardella — APPROVED** [review]

**Pull request merged**